### PR TITLE
Spanish translation.json

### PR DESCRIPTION
**Pull request by mmax2001**
Spanish translate

### CODE REVIEW

**Comment by mmax2001** [comment]
Add Spanish translation for mintr

**Comment by 0xclem** [comment]
@mmax2001 thanks! Can you please rename the `sp` folder to `es` to follow the ISO 639-1 standards?

Then we'll review the translation and merge it if everything is ok.

File: synthetix-mintr/sp/translation.json [hunk 1]
```diff
@@ -0,0 +1,409 @@
+{
```
Comment (0xclem) [inline synthetix-mintr/sp/translation.json]: You can now delete this file. Git still thinks there's a `sp/translation.json`

**Comment by mmax2001** [comment]
Yes of course!

**Review by 0xclem — CHANGES_REQUESTED** [review]

**Comment by 0xclem** [comment]
@mmax2001 Sorry I should have mentioned how to do it. 

Can you please remove the changes in the `gitignore` as we don't want to edit this file.

If you have git installed in command line you can try something like `git rm sp/translation.json` instead. If still no luck don't worry I'll take care of that.

**Comment by mmax2001** [comment]
Ok, no problem!

**Comment by 0xclem** [comment]
Hey @mmax2001 , thanks for your PR. Because of the next release we had to make some minor changes to the initial EN json file. Would you mind fixing your PR so it can use the latest updates?

Easiest way would be to start again from the new en/translation.json file as its structure changed a bit. Sorry again for the inconvenience.

FYI, here are the changes: https://github.com/Synthetixio/synthetix-translations/pull/3/files#diff-6000b66f2ed7c8f8c4e193a38d780897

**Comment by mmax2001** [comment]
> Hey @mmax2001 , thanks for your PR. Because of the next release we had to make some minor changes to the initial EN json file. Would you mind fixing your PR so it can use the latest updates?
> 
> Easiest way would be to start again from the new en/translation.json file as its structure changed a bit. Sorry again for the inconvenience.
> 
> FYI, here are the changes: https://github.com/Synthetixio/synthetix-translations/pull/3/files#diff-6000b66f2ed7c8f8c4e193a38d780897

New translatation is uploaded

**Review by 0xclem — CHANGES_REQUESTED** [review]
@mmax2001 Keep `node_modules`. `.gitignore` shouldn't be changed at all in the PR.

**Comment by mmax2001** [comment]
> shouldn't be changed at all in the PR.

how to keep these files in my fork?

**Comment by 0xclem** [comment]
Thanks @mmax2001. Unfortunately there is now 7 files committed in the PR instead of 1, so I'll just close it and do a release with your translation file. Will keep you updated!

**Comment by mmax2001** [comment]
Oh I’m sorry! 
Thanks in advance!

**Comment by mmax2001** [comment]
Hi Clément! I found a minor few issues in the translate, so I have fixed it .

**Comment by 0xclem** [comment]
Hi @mmax2001, thanks! Can you please create a new Pull request with your change? Make sure you only commit one file :)

**Pull request closed without merging**